go.mod

```diff
@@ -3,8 +3,27 @@ module github.com/container-orchestrated-devices/container-device-interface
 go 1.14
 
 require (
+	github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6 // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/coreos/etcd v3.3.10+incompatible // indirect
+	github.com/coreos/go-etcd v2.0.0+incompatible // indirect
+	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/fsnotify/fsnotify v1.5.1 // indirect
+	github.com/hashicorp/go-multierror v1.1.1 // indirect
+	github.com/mitchellh/go-homedir v1.1.0 // indirect
 	github.com/opencontainers/runtime-spec v1.0.2
```

Comment:
We run the `Unmarshal()`ler now in strict mode. So I think that's as good/bad now as it can reasonably get. I tried with the current monitor code and this is what I get if I add `"anUnknownField": true` in there:
```
klitkey1-mobl container-device-interface $ ./bin/cdi monitor --spec-dirs $(pwd) -v -oyaml all
Spec Files in CDI Registry:
*** CDI Registry has errors:
  failed to load CDI Spec "/home/kli/src/work/cri/container-device-interface/test-devices.json": failed to parse CDI Spec: error unmarshaling JSON: while decoding JSON: json: unknown field "anUnknownField"
No vendors found in CDI registry.
No CDI device classes found.
No CDI Devices found.
```

Reply:
Ok. Great. We will also switch to strict mode on our side for the time being.